README.md

```diff
@@ -220,6 +220,8 @@ For efficient searches we recommend to install `pt` ([the platinum searcher][]).
 
    You may wish to clear out your `.emacs.d/elpa` directory before doing this,
    so that any corrupted packages you may have downloaded will be reinstalled.
+   Or you may need to set the `dotspacemacs-elpa-https` to `nil` in dotfiles  
```

Comment:
I think this should go before

```
    You may wish to clear out your `.emacs.d/elpa` directory before doing this,
     so that any corrupted packages you may have downloaded will be reinstalled.
```

Because it's related to disabling HHTPS protocol. When quoted advice is for running after you disabled HTTPS protocol.